src/Model/InlineObject.php

```diff
@@ -1,55 +1,311 @@
 <?php
-/**
- * This source file is proprietary and part of Rebilly.
- *
- * (c) Rebilly SRL
- *     Rebilly Ltd.
- *     Rebilly Inc.
- *
- * @see https://www.rebilly.com
- */
 
 declare(strict_types=1);
 
 namespace Rebilly\Sdk\Model;
 
 use JsonSerializable;
+use InvalidArgumentException;
+use TypeError;
 
-class InlineObject implements JsonSerializable
-{
-    private array $fields = [];
-
+class InlineObject {
```

Comment:
This is a strange name, no?

Reply:
Not sure how the auto generation works, but seems that the interface was removed here. Maybe who have worked with auto generation can comment on it?

Reply:
It is a fallback name for objects without a separate schema name or title. PatchCreditMemo operation definition must be fixed.